metro.config.js

```diff
@@ -1,16 +0,0 @@
-const { getDefaultConfig } = require("metro-config");
-
-module.exports = (async () => {
-  const {
-    resolver: { sourceExts, assetExts }
-  } = await getDefaultConfig();
-  return {
-    transformer: {
-      babelTransformerPath: require.resolve("react-native-svg-transformer")
```

Comment:
I found no `.svg` file imports on any of the solutions. Is it still perhaps an intended part of the "work in progress" and therefore something we want to support?

Reply:
If I remember correctly, I used this while I was drawing the SVGs to be able to preview them on Slimy. But I think we can remove it, as there is no real need to have this library — there are many other SVG external tools that can serve the same purpose.

Reply:
Answered on the other thread.